frontend/src/app/reducers/browser.js

```diff
@@ -8,7 +8,11 @@ const isUserAgentMobile = isMobile(userAgent);
 const isUserAgentMinFirefox = isMinFirefoxVersion(userAgent, config.minFirefoxVersion);
 const locale = (navigator.language || '').split('-')[0];
 
-export default handleActions({}, {
+export const setState = (state) => state;
```

Comment:
It feels to me like this should be importing and using the action from the other module, but I can't say I really understand redux-actions.